dropwizard-archetypes/java-simple/src/main/resources/archetype-resources/src/main/java/__name__Application.java

```diff
@@ -3,16 +3,20 @@
 import io.dropwizard.Application;
 import io.dropwizard.setup.Bootstrap;
 import io.dropwizard.setup.Environment;
+import ${package}.health.${name}HealthCheck;
+import ${package}.resources.${name}Resource;
 
 public class ${name}Application extends Application<${name}Configuration> {
 
+    private String applicationName;
+
     public static void main(final String[] args) throws Exception {
         new ${name}Application().run(args);
     }
 
     @Override
     public String getName() {
-        return "${name}";
```

Comment:
I prefer the previous approach more as it was easier to understand and more standard.